pyzo/core/editor.py

```diff
@@ -259,6 +259,7 @@ def __init__(self, parent, **kwds):
         self.setWrap(bool(pyzo.config.view.wrap))
         self.setHighlightCurrentLine(pyzo.config.view.highlightCurrentLine)
         self.setLongLineIndicatorPosition(pyzo.config.view.edgeColumn)
+        self.setHighlightMatchingBracket(pyzo.config.view.highlightMatchingBracket)
```

Comment:
Yes, I added a line in my config file. Can you take care of these things relating to options/menus ? I have not read that part of the code and have no idea how it works (and do not know Qt either).
I added a method highlightMisMatchingBracket in the class for tweaking whether to highlight the red parentheses in my example ; maybe you will want to expose it to the option system too.